org.eclipse.tm4e.ui/src/main/java/org/eclipse/tm4e/ui/internal/themes/DefaultThemeBehaviour.java

```diff
@@ -0,0 +1,20 @@
+package org.eclipse.tm4e.ui.internal.themes;

```

Comment:
Why not using your extension point to register DefaultThemeBehaviour ?

Reply:
I'd prefer to always have some default behaviour. I think that if we will register it, ThemeBehaviourManager.init() should change to 
`while (behaviour == null && configurationElements.length > 0) {
behaviour = getBehaviour(configurationElements[i]);
i++;
}` or something like that, where we occure exception and we wont have any behaviour.